shard-operator-parent/shard-operator-v2/src/main/java/com/redhat/service/smartevents/shard/operator/v2/reconcilers/ManagedBridgeReconciler.java

```diff
@@ -0,0 +1,67 @@
+package com.redhat.service.smartevents.shard.operator.v2.reconcilers;
+
+import com.redhat.service.smartevents.shard.operator.v2.DeltaProcessorService;
+import com.redhat.service.smartevents.shard.operator.v2.ManagedBridgeService;
+import com.redhat.service.smartevents.shard.operator.v2.ManagerClient;
+import com.redhat.service.smartevents.shard.operator.v2.comparators.Comparator;
+import com.redhat.service.smartevents.shard.operator.v2.comparators.ManagedBridgeComparator;
+import com.redhat.service.smartevents.shard.operator.v2.converters.ManagedBridgeConverter;
+import com.redhat.service.smartevents.shard.operator.v2.resources.ManagedBridge;
+import io.smallrye.mutiny.Uni;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.inject.Named;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@ApplicationScoped
+public class ManagedBridgeReconciler {
```

Comment:
@vaibhavjainwiz To get us started an minimise the changes, how about this calls the `ManagedBridgeService` directly with the correct create, update or delete method.

I'm not against your delta approach, however I'd really like to get the Bridge flow working without making too many changes to it. Once we have it working, we can then decide how best to decompose it further. For example I am seeing many other improvements we can make to support code re-use, but my main focus right now is on getting things working.

WDYT?

Reply:
done.. I had made the requested changes.